talk/objectorientation.tex

```diff
@@ -132,7 +132,7 @@
     \begin{itemize}
     \item overloading is authorized and welcome
     \item signature is part of the method identity
-    \item but not the return code
+    \item but not the return type
```

Comment:
No I really mean type here. This should not be changed.

Reply:
```suggestion
    \item but not the return code
```

Reply:
> No I really mean >>type<< here. This should not be changed.

Was this a typo in the above message? So we use code like "exit code".

Reply:
Yes typo... now fixed It's really the type that is important here. I'm not sure actually what you meant by "code"

Reply:
OK, agree on using "return type".